### ISSUE TITLE
[P2][Sourcing][Accessibility][Multi input list]The status of the list entries is not perceptible with the screen reader (aria-selected=true or false) because aria-multiselectable is missing.

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10289]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
The status of the list entries is not perceptible with the screen reader (aria-selected=true or false) because aria-multiselectable is missing.

#### If this is a bug, please provide steps for reproducing it.
The status of the list entries is not perceptible with the screen reader (aria-selected=true or false) because aria-multiselectable is missing.
tested on multi input mobile mode example

#### Is there anything else we should know?
reported by customer

**Comment by I543348** [comment]
ATL comment:
Screen reader users cannot differentiate between the selected list items and normal list items within the value help. Here information “Selected” is not being read for the selected list items.

**Comment by nikolay-kolarov** [comment]
Hi,

Not sure if only the aria-multiselectable will help, as I think the screen reader does not announce the selected state for option in listbox. In UI5 there is a custom string "selected" added on focus over a selected list item: https://ui5.sap.com/#/entity/sap.m.MultiComboBox/sample/sap.m.sample.MultiComboBox

Best Regards,
Nikolay

**Comment by droshev** [comment]
We need to downport this to `ng15`

**Comment by mikerodonnell89** [comment]
Possibly fixed by #10433 however I am seeing mac VoiceOver is reading the items as selected/not selected when focused. @I543348 what screenreader are you referring to?

**Comment by droshev** [comment]
fixed with `0.43.7`

**Comment by sagar7259** [comment]
Hello,

Initially on selecting any list items, its corresponding information "Selected" is being read to the users. But after selection when user uses arrow keys for the navigation, all the list items are read in the same way.

Example: There are 3 list items apple, mango, orange in the list and user selects apple where the information is read as "Apple 1 of 3 selected".
Later when user uses arrow keys to navigate within all 3 list items the reading is made has:
" Apple 1 of 3
Mango 2 of 3
Orange 3 of 3"

Here the apple is already selected, but JAWS does not read this has selected to the users.

@rvamudha please reopen this ticket.

Regards,
Sagar

**Comment by droshev** [comment]
If you tested Multi Input Mobile - this is not supposed to be testing on desktop. It is just to demonstrate how the component will behave on Mobile(not on desktop)

**Comment by sagar7259** [comment]
@droshev my previous reply holds true for the desktop example also.
You can verify this with the desktop example: https://sap.github.io/fundamental-ngx/#/core/multi-input

Here is the screenshot and explination of the issue,

apple, banana and pineapple are the selected tokens but when you look into the speech history on the right side,,, no information about selected is being read.
![image](https://github.com/SAP/fundamental-ngx/assets/143692208/b4c5227a-79b6-467c-bd8d-ec257a6a4e7c)

@rvamudha please reopen this ticket.

Regards,
Sagar

**Comment by droshev** [comment]
you need to test against version `0.43.10`

**Comment by sagar7259** [comment]
Hi Team,

This issue is now fixed as per the verification against version 0.43.10, hence this ticket can be closed.

Regards,
Sagar
